src/components/page-details.jsx

```diff
@@ -226,13 +226,29 @@ export default class PageDetails extends React.Component {
     // TODO: handle the missing `.versions` collection problem better
     const fromList = this.props.pages && this.props.pages.find(
       (page) => page.uuid === pageId && !!page.versions);
-
+    
+    /** HACK: To deal with the huge number of versions coming from Internet Archive,
+     * we're returning only versions captured after November 1, 2016 until we figure out a 
+     * better solution. Probably an improved iteration of timeline idea:
+     * https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/98
+     * Issue outlined here: https://github.com/edgi-govdata-archiving/web-monitoring-db/issues/264
+     */
+    const cutoffDate = '2016-11-01';
```

Comment:
Part of the importance of making this a constant is to remove it from the middle of a giant pile of code and put it either at the top of the file or somewhere in the constants directory, so it’s easy to see/find and modify.